### PR TITLE
[Doc Feature][Call recording API][3099832]

### PR DESCRIPTION
**Pull request by v-sdhakshina**

### CODE REVIEW

**Review by devjha-ms — COMMENTED** [review]
Changes looks good, few comments.

File: api-reference/beta/api/callrecording-get.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+#### Response
+
+Response contains bytes for the recording in the body. `content-type` header specifies type of the recording content. Negative offsets indicate that the recording on began while the conversation was ongoing.
+
+>**Note:** The response object shown here might be shortened for readability.
+
+```http
+
+```
```
Comment (devjha-ms) [inline api-reference/beta/api/callrecording-get.md:124]: remove this extra http code.

**Comment by devjha-ms** [comment]
Looks good

**Review by devjha-ms — APPROVED** [review]
Looks good.

File: api-reference/beta/api/onlinemeeting-list-recordings.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+> - In the future, Microsoft may require you or your customers to pay additional fees based on the amount of data accessed through the API.
+> - This API works differently in one or more national clouds. For details, see [Implementation differences in national clouds](/graph/teamwork-national-cloud-differences).
+
+## Permissions
+
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | OnlineMeetingRecording.Read.All |
```
[the remark this answers is not in the document]
Comment (v-sdhakshina) [reply api-reference/beta/api/onlinemeeting-list-recordings.md:29]: updated as suggested

File: api-reference/beta/api/callrecording-get.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+>**Note:** The response object shown here might be shortened for readability.
+
+```http
+HTTP/1.1 200 OK
+Content-type: video/mp4
+
+WEBVTT
+
+0:0:0.0 --> 0:0:5.320
+<v User Name>This is a recording test.</v>
```
[the remark this answers is not in the document]
Comment (v-sdhakshina) [reply api-reference/beta/api/callrecording-get.md]: Updated as suggested

File: api-reference/beta/api/callrecording-get.md (hunk 1 above)
Comment (v-sdhakshina) [reply api-reference/beta/api/callrecording-get.md:124]: As discussed with Devi, kept this tag as it is. Content reviewer needs to provide input for this.

File: api-reference/beta/api/onlinemeeting-list-recordings.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+> [!NOTE]
+> This API works for a meeting only if the meeting has not expired. For more details, see [Limits and specifications for Microsoft Teams](/microsoftteams/limits-specifications-teams#meeting-expiration).
+
+## HTTP request
+
+```http
+GET /me/onlineMeetings({meetingId})/recordings
+GET /users({userId})/onlineMeetings({meetingId})/recordings
+```
```
Comment (ayan-ms) [inline api-reference/beta/api/onlinemeeting-list-recordings.md:46]: Seems few forward slashes are missing here. It should be /me/onlineMeetings**/{meetingId}/**recordings. Similarly /users**/{userId}/**onlineMeetings...

File: api-reference/beta/api/onlinemeeting-list-recordings.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Retrieve the list of [callrecording](../resources/callrecording.md) objects associated with an [onlineMeeting](../resources/onlinemeeting.md).
+
+> **Notes:**
+
+> - In the future, Microsoft may require you or your customers to pay additional fees based on the amount of data accessed through the API.
+> - This API works differently in one or more national clouds. For details, see [Implementation differences in national clouds](/graph/teamwork-national-cloud-differences).
```
Comment (ayan-ms) [inline api-reference/beta/api/onlinemeeting-list-recordings.md:21]: Please add a section in page [Implementation differences in national clouds] (https://review.learn.microsoft.com/en-us/graph/teamwork-national-cloud-differences?branch=pr-en-us-21546) stating that "Meeting recording APIs are not supported" with content same as section for transcripts with minor changes linking to recording pages.

**Review by ayan-ms — CHANGES_REQUESTED** [review]
Reviewed this PR

**Comment by v-sdhakshina** [comment]
Closing this PR as it has added too many files.

**Pull request closed without merging**